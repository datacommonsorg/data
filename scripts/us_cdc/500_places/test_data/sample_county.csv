Year,StateAbbr,StateDesc,LocationName,DataSource,Category,Measure,Data_Value_Unit,Data_Value_Type,Data_Value,Data_Value_Footnote_Symbol,Data_Value_Footnote,Low_Confidence_Limit,High_Confidence_Limit,TotalPopulation,Latitude,Longitude,LocationID,CategoryID,MeasureId,DataValueTypeID,Short_Question_Text,geolocation
2018,US,United States,,BRFSS,Prevention,Current lack of health insurance among adults aged 18-64 years,%,Crude prevalence,12.2,,,11.9,12.4,327167434,,,59,PREVENT,ACCESS2,CrdPrv,Health Insurance,
2018,AL,Alabama,Bibb,BRFSS,Health Outcomes,Cancer (excluding skin cancer) among adults aged >=18 years,%,Crude prevalence,7.2,,,6.9,7.5,22400,32.99854412,-87.12653219,01007,HLTHOUT,CANCER,CrdPrv,Cancer (except skin),POINT (-87.12653219 32.99854412)
2018,AL,Alabama,Bullock,BRFSS,Health Outcomes,Cancer (excluding skin cancer) among adults aged >=18 years,%,Age-adjusted prevalence,5.8,,,5.6,6.1,10138,32.10044482,-85.71563262,01011,HLTHOUT,CANCER,AgeAdjPrv,Cancer (except skin),POINT (-85.71563262 32.10044482)
2018,AL,Alabama,Choctaw,BRFSS,Health Outcomes,Current asthma among adults aged >=18 years,%,Crude prevalence,11.4,,,10.8,12.0,12841,32.020526,-88.26304964,01023,HLTHOUT,CASTHMA,CrdPrv,Current Asthma,POINT (-88.26304964 32.020526)
2018,AL,Alabama,Cleburne,BRFSS,Unhealthy Behaviors,Binge drinking among adults aged >=18 years,%,Crude prevalence,13.3,,,12.5,14.0,14987,33.67451857,-85.51894991,01029,UNHBEH,BINGE,CrdPrv,Binge Drinking,POINT (-85.51894991 33.67451857)
2018,AL,Alabama,Cleburne,BRFSS,Unhealthy Behaviors,Obesity among adults aged >=18 years,%,Crude prevalence,37.1,,,35.3,38.8,14987,33.67451857,-85.51894991,01029,UNHBEH,OBESITY,CrdPrv,Obesity,POINT (-85.51894991 33.67451857)
2018,AL,Alabama,Dale,BRFSS,Health Outcomes,Arthritis among adults aged >=18 years,%,Age-adjusted prevalence,31.2,,,30.0,32.3,48956,31.43150889,-85.61159073,01045,HLTHOUT,ARTHRITIS,AgeAdjPrv,Arthritis,POINT (-85.61159073 31.43150889)
2018,AL,Alabama,Dallas,BRFSS,Health Outcomes,Arthritis among adults aged >=18 years,%,Crude prevalence,36.8,,,35.5,38.0,38310,32.32602481,-87.1066303,01047,HLTHOUT,ARTHRITIS,CrdPrv,Arthritis,POINT (-87.1066303 32.32602481)
2017,AL,Alabama,Elmore,BRFSS,Prevention,Taking medicine for high blood pressure control among adults aged >=18 years with high blood pressure,%,Crude prevalence,77.6,,,76.7,78.6,81887,32.59685888,-86.14875636,01051,PREVENT,BPMED,CrdPrv,Taking BP Medication,POINT (-86.14875636 32.59685888)
