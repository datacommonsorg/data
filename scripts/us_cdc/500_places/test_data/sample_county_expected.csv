Year,StateAbbr,StateDesc,LocationName,DataSource,Category,Measure,Data_Value_Unit,Data_Value_Type,Data_Value,Data_Value_Footnote_Symbol,Data_Value_Footnote,Low_Confidence_Limit,High_Confidence_Limit,TotalPopulation,Latitude,Longitude,LocationID,CategoryID,MeasureId,DataValueTypeID,Short_Question_Text,geolocation,Low_Confidence_Limit_StatVar,High_Confidence_Limit_StatVar,Population_StatVar,ConfidenceLevelTypeID
2018,US,United States,,BRFSS,Prevention,Current lack of health insurance among adults aged 18-64 years,%,Crude prevalence,12.2,,,11.9,12.4,327167434,,,59,PREVENT,dcs:Percent_Person_18To64Years_ReceivedNoHealthInsurance,dcs:CrudePrevalence,Health Insurance,,dcs:LowerConfidenceIntervalLimit_Percent_Person_18To64Years_ReceivedNoHealthInsurance,dcs:UpperConfidenceIntervalLimit_Percent_Person_18To64Years_ReceivedNoHealthInsurance,dcs:SampleSize_Percent_Person_18To64Years_ReceivedNoHealthInsurance,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Bibb,BRFSS,Health Outcomes,Cancer (excluding skin cancer) among adults aged >=18 years,%,Crude prevalence,7.2,,,6.9,7.5,22400,32.99854412,-87.12653219,1007,HLTHOUT,dcs:Percent_Person_WithCancerExcludingSkinCancer,dcs:CrudePrevalence,Cancer (except skin),POINT (-87.12653219 32.99854412),dcs:LowerConfidenceIntervalLimit_Percent_Person_WithCancerExcludingSkinCancer,dcs:UpperConfidenceIntervalLimit_Percent_Person_WithCancerExcludingSkinCancer,dcs:SampleSize_Percent_Person_WithCancerExcludingSkinCancer,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Bullock,BRFSS,Health Outcomes,Cancer (excluding skin cancer) among adults aged >=18 years,%,Age-adjusted prevalence,5.8,,,5.6,6.1,10138,32.10044482,-85.71563262,1011,HLTHOUT,dcs:Percent_Person_WithCancerExcludingSkinCancer,dcs:AgeAdjustedPrevalence,Cancer (except skin),POINT (-85.71563262 32.10044482),dcs:LowerConfidenceIntervalLimit_Percent_Person_WithCancerExcludingSkinCancer,dcs:UpperConfidenceIntervalLimit_Percent_Person_WithCancerExcludingSkinCancer,dcs:SampleSize_Percent_Person_WithCancerExcludingSkinCancer,dcs:AgeAdjustedPrevalenceConfidenceLevel
2018,AL,Alabama,Choctaw,BRFSS,Health Outcomes,Current asthma among adults aged >=18 years,%,Crude prevalence,11.4,,,10.8,12.0,12841,32.020526000000004,-88.26304964,1023,HLTHOUT,dcs:Percent_Person_WithAsthma,dcs:CrudePrevalence,Current Asthma,POINT (-88.26304964 32.020526),dcs:LowerConfidenceIntervalLimit_Percent_Person_WithAsthma,dcs:UpperConfidenceIntervalLimit_Percent_Person_WithAsthma,dcs:SampleSize_Percent_Person_WithAsthma,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Cleburne,BRFSS,Unhealthy Behaviors,Binge drinking among adults aged >=18 years,%,Crude prevalence,13.3,,,12.5,14.0,14987,33.67451857,-85.51894991,1029,UNHBEH,dcs:Percent_Person_BingeDrinking,dcs:CrudePrevalence,Binge Drinking,POINT (-85.51894991 33.67451857),dcs:LowerConfidenceIntervalLimit_Percent_Person_BingeDrinking,dcs:UpperConfidenceIntervalLimit_Percent_Person_BingeDrinking,dcs:SampleSize_Percent_Person_BingeDrinking,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Cleburne,BRFSS,Unhealthy Behaviors,Obesity among adults aged >=18 years,%,Crude prevalence,37.1,,,35.3,38.8,14987,33.67451857,-85.51894991,1029,UNHBEH,dcs:Percent_Person_Obesity,dcs:CrudePrevalence,Obesity,POINT (-85.51894991 33.67451857),dcs:LowerConfidenceIntervalLimit_Percent_Person_Obesity,dcs:UpperConfidenceIntervalLimit_Percent_Person_Obesity,dcs:SampleSize_Percent_Person_Obesity,dcs:CrudePrevalenceConfidenceLevel
2018,AL,Alabama,Dale,BRFSS,Health Outcomes,Arthritis among adults aged >=18 years,%,Age-adjusted prevalence,31.2,,,30.0,32.3,48956,31.43150889,-85.61159073,1045,HLTHOUT,dcs:Percent_Person_WithArthritis,dcs:AgeAdjustedPrevalence,Arthritis,POINT (-85.61159073 31.43150889),dcs:LowerConfidenceIntervalLimit_Percent_Person_WithArthritis,dcs:UpperConfidenceIntervalLimit_Percent_Person_WithArthritis,dcs:SampleSize_Percent_Person_WithArthritis,dcs:AgeAdjustedPrevalenceConfidenceLevel
2018,AL,Alabama,Dallas,BRFSS,Health Outcomes,Arthritis among adults aged >=18 years,%,Crude prevalence,36.8,,,35.5,38.0,38310,32.32602481,-87.10663029999999,1047,HLTHOUT,dcs:Percent_Person_WithArthritis,dcs:CrudePrevalence,Arthritis,POINT (-87.1066303 32.32602481),dcs:LowerConfidenceIntervalLimit_Percent_Person_WithArthritis,dcs:UpperConfidenceIntervalLimit_Percent_Person_WithArthritis,dcs:SampleSize_Percent_Person_WithArthritis,dcs:CrudePrevalenceConfidenceLevel
2017,AL,Alabama,Elmore,BRFSS,Prevention,Taking medicine for high blood pressure control among adults aged >=18 years with high blood pressure,%,Crude prevalence,77.6,,,76.7,78.6,81887,32.59685888,-86.14875636,1051,PREVENT,dcs:Percent_Person_18OrMoreYears_WithHighBloodPressure_ReceivedTakingBloodPressureMedication,dcs:CrudePrevalence,Taking BP Medication,POINT (-86.14875636 32.59685888),dcs:LowerConfidenceIntervalLimit_Percent_Person_18OrMoreYears_WithHighBloodPressure_ReceivedTakingBloodPressureMedication,dcs:UpperConfidenceIntervalLimit_Percent_Person_18OrMoreYears_WithHighBloodPressure_ReceivedTakingBloodPressureMedication,dcs:SampleSize_Percent_Person_18OrMoreYears_WithHighBloodPressure_ReceivedTakingBloodPressureMedication,dcs:CrudePrevalenceConfidenceLevel
